indico/modules/designer/templates/template.html

```diff
@@ -370,7 +377,7 @@
             'use strict';
 
             setupDesigner({{ template_data | tojson }}, {{ backside_template_data | tojson}}, {{ config | tojson }},
```

Comment:
```suggestion
            setupDesigner({{ template_data | tojson }}, {{ backside_template_data | tojson }}, {{ config | tojson }},
```

(not related to your change, just noticed it)